crates/e2e/tests/e2e/cow_amm.rs

```diff
@@ -408,9 +408,13 @@ async fn forked_node_mainnet_cow_amm_driver_support() {
         cow_amm_driver_support,
         std::env::var("FORK_URL_MAINNET")
             .expect("FORK_URL_MAINNET must be set to run forked tests"),
-        // block at which helper was deployed, this block can't be updated since it would lead to
-        // the long indexing problem and, as a result, failing tests
-        20332745,
+        // There are 2 types of CoW AMM pool creation events:
```

Comment:
This is a really useful comment to have but having it here in some e2e test seems odd. Here I would probably just say sth like "block before relevant cow amm was finalized".